q2_composition/_ancombc.py

```diff
@@ -0,0 +1,159 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2022, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+import subprocess
+import tempfile
+import pandas as pd
+import os
+import formulaic
+
+import qiime2
+
+from q2_stats._format import DataLoafPackageDirFmt
+
+
+def run_commands(cmds, verbose=True):
```

Comment:
Yeah that's definitely a good point - @ebolyen what do you think about a qiime2/utils little helper method for this functionality?

Reply:
Circling back on this for posterity - @ebolyen and I talked about this, and this would be great to add in the framework sometime in the future, but we'll leave as-is for now. I will add this as an issue in our rainy day project board 🙂